libc/include/unistd.yaml

```diff
@@ -128,6 +128,13 @@ functions:
     arguments:
       - type: char *
       - type: size_t
+  - name: getentropy
+    standards:
+      - GNUExtensions
+    return_type: int
+    arguments:
+      - type: void *
+      - type: size_t
```

Comment:
The man page has a note:

> getentropy() is also declared in <sys/random.h>

For compatibility, should you add another declaration there?